projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/active-integrations/index.js

```diff
@@ -0,0 +1,48 @@
+import colorStudio from '@automattic/color-studio';
+import { JetpackIcon } from '@automattic/jetpack-components';
+import { Spinner } from '@wordpress/components';
+import AkismetIcon from '../../../../../icons/akismet-icon';
+import './style.scss';
+
+const COLOR_JETPACK = colorStudio.colors[ 'Jetpack Green 40' ];
+
+export default function ActiveIntegrations( { integrations, attributes, isLoading } ) {
+	const activeIntegrations = integrations.reduce( ( acc, integration ) => {
+		switch ( integration.id ) {
+			case 'akismet':
+				if ( integration.isConnected ) {
+					acc.push( { ...integration, icon: <AkismetIcon width={ 30 } height={ 30 } /> } );
+				}
+				break;
+			case 'zero-bs-crm':
+				if ( integration.isActive && integration.details?.hasExtension && attributes.jetpackCRM ) {
+					acc.push( { ...integration, icon: <JetpackIcon size={ 30 } color={ COLOR_JETPACK } /> } );
+				}
+				break;
+		}
+		return acc;
+	}, [] );
```

Comment:
The reduce method takes in the list of integrations, and returns: 
(a) just those that are active/enabled for this form and
(b) includes the icon so it can be output below